src/components/MonacoEditor/index.tsx

```diff
@@ -46,11 +58,16 @@ export const MonacoEditor = () => {
           {
             fileMatch: ["*"],
             schema: jsonSchema,
+            uri: "",
           },
         ],
       }),
     });
```

Comment:
What is the purpose of this change here?

Reply:
This was to solve a type issue after I referenced `monaco-editor` types. Removing it to keep this PR more scoped. 

![Screenshot 2023-09-17 at 23 10 58](https://github.com/AykutSarac/jsoncrack.com/assets/630796/4d4beda6-4345-47a8-9116-9d6a28974920)